answer_files/2008_r2/Autounattend.xml

```diff
@@ -232,16 +232,15 @@
                     <Description>Disable password expiration for vagrant user</Description>
```

Comment:
May want to change this line:
`<CommandLine>cmd.exe /c winrm set winrm/config/winrs @{MaxMemoryPerShellMB="800"}</CommandLine>`

To be
`<CommandLine>cmd.exe /c winrm set winrm/config/winrs @{MaxMemoryPerShellMB="2048"}</CommandLine>`

To closer match the system image we bake